src/app/routines/routineCard/RoutineCard.tsx

```diff
@@ -0,0 +1,45 @@
+import MoreVertIcon from "@mui/icons-material/MoreVert";
+import ThumbUpOffAltIcon from "@mui/icons-material/ThumbUpOffAlt";
+import React from "react";
+
+import "./RoutineCard.scss";
+
+const RoutineCard: React.FC<Props> = (props) => {
+  return (
+    <div className="routine-list-item">
+      <div className="routine-list-item_name">
+        <h3>
+          <title>{props.name}</title>
+          {props.name}
+        </h3>
+      </div>
+      <div className="routine-list-item_description">
+        <p>{props.description}</p>
+      </div>
+      <div className="routine-list-item_options">
+        <MoreVertIcon />
+      </div>
+      <div className="likes-container">
+        <div className="likes-number">
+          <p>
+            {props.likes > 999
```

Comment:
better to handle that logic in function outside and even handle memoization of the value

const likes = useMemo(()=>{
props.likes > 999 ? (props.likes / 1000).toFixed(0).concat("k") : props.likes
},[props.likes]);

<p>{likes}</p>

Reply:
ok, but likes is a number, so it is impossible to save it as a string :/

Reply:
if you want to return string out of number its pretty simple, you can always do:

`{numberVariable}`

And adding that useMemo optimizes ur code, cause otherwise with each rerender that function will calculate it's value, if you do useMemo, it will store that value in browser cache and if at the end you do [variable], it will recalculate that if the variable actually changed otherwise it will returned cached data

Reply:
Ok I thought that mistake is somewhere else, there should be a "return" word in useMemo function, now it's ok